daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -59,10 +59,10 @@ You can find information [here]({{< ref "install-dapr.md#using-helm-advanced" >}
 When deploying Dapr in a production-ready configuration, it's recommended to deploy with a highly available configuration of the control plane:
 
 ```bash
-helm install dapr dapr/dapr --namespace dapr-system --set global.ha.enabled=true
+helm install dapr dapr/dapr --version=1.0.0-rc.x --namespace dapr-system --set global.ha.enabled=true
```

Comment:
Should this specific command apply only to rc.x releases?

Reply:
It shouldn't..

Reply:
@yaron2 @orizohar / I have talked to ori offline. Without chart version option, user will get 0.11.3 dapr, but doc's master repo will have the information for 1.0.0-rc.x, not for 0.11.3.

To reduce the confusion, we may need to ask users to specify the specific version of chart.

I will change 1.0.0-rc.x to `<Dapr runtime version>` to let user specify the specific version of dapr.